app/package.json

```diff
@@ -12,56 +12,43 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.15.7",
-    "@react-native-masked-view/masked-view": "^0.2.6",
     "@react-navigation/bottom-tabs": "^6.0.5",
     "@react-navigation/native": "^6.0.2",
-    "@react-navigation/stack": "^5.x",
+    "@react-navigation/stack": "^6.x",
     "@reduxjs/toolkit": "^1.6.1",
-    "@thecodingmachine/redux-toolkit-wrapper": "2.0.1",
-    "axios": "^0.21.4",
     "dayjs": "^1.11.1",
     "fuse.js": "^6.6.2",
     "i18next": "^20.6.1",
-    "prop-types": "^15.7.2",
-    "react": "17.0.2",
+    "moment": "^2.29.3",
```

Comment:
The date picker uses it. Not sure why it's not in package.json but it should be. Might have had it installed globally on mine maybe.

Reply:
The date picker uses it. Not sure why it's not in package.json but it should be. Might have had it installed globally on mine maybe.

Reply:
Ah right, thanks.  (It's weird that package doesn't just require it as a dependency rather than us having to install it manually, but I can see that's in their instructions.)